fluent-bit-cloudwatch.go

```diff
@@ -121,6 +121,9 @@ func getConfiguration(ctx unsafe.Pointer, pluginID int) cloudwatch.OutputPluginC
 	config.LogFormat = output.FLBPluginConfigKey(ctx, "log_format")
 	logrus.Infof("[cloudwatch %d] plugin parameter log_format = '%s'", pluginID, config.LogFormat)
 
+	config.ExtraUserAgent = output.FLBPluginConfigKey(ctx, "extra_user_agent")
+	logrus.Infof("[cloudwatch %d] plugin parameter extra_user_agent = '%s'", pluginID, config.ExtraUserAgent)
```

Comment:
See my comment on the readme- let's not print the value of this field. Remove the info. We want it to be slightly hidden.

Reply:
Will update this part later